barrels/README.md

```diff
@@ -0,0 +1,10 @@
+# Barrels
+A collection of Connect IQ libraries.
+
+## Semicircles
+This barrel provides an abstract coordinate type that internally uses a semicircles representation.
+The Coordinate class does not use Position module and does not require the Position permission.
+
+This class uses 32 bit semicircles, where PI radians are equal to 0x80000000 32 bit semicircles.
+Semicircles handle wrap-around via integer overflow, and all angle math can be done using integer math
+speeding up computation. This class also includes a fast distance between two point computation.
```

Comment:
Move the Semicircles block to it's own README.md within the Semicircles folder. Also maybe include a blip about what semicircles are. Working for Garmin I know of semicircles but having never really worked on product code I have a poor understanding of what they actually are. I suspect other developers will have this same gap of knowledge.

Reply:
Done